src/transformState.js

```diff
@@ -6,6 +6,41 @@
  */
 function transformState(state, actions) {
   // write code here
+  function addProperties(extraData) {
+    for (const [key, value] of Object.entries(extraData)) {
+      state[key] = value;
+    }
```

Comment:
Better use [Object.assign](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) here, to avoid nested loop